### ISSUE TITLE
Category in mint and explorer

### ISSUE THREAD

**Issue by exezbcz** [kodadot/nft-gallery#8032]
## Abstract 
- adding a category to minting 
- category filtering to explorer 
- categories are predefined, one selected only
- updated in figma handoff: link

List of categories: 
- Gaming
- Generative
- Digital Art
- Music & Audio
- Photography
- 3D models
- AI-Generated
- NSFW

# Design changes

### Sections in mint 
- this is for mint nft
![image](https://github.com/kodadot/nft-gallery/assets/90852205/a6314479-3b75-43dc-bbcb-c76168f3463a)
- displayed as the last section, after nsfw 
- there is a toggle button to inherit the category set for the collection 
- note that there is also a tooltip

in the case of minting collection
- last position as well with different copy
![image](https://github.com/kodadot/nft-gallery/assets/90852205/b86a7727-8fa8-4182-beec-465d00f3340c)

## Explore
now for the explore filters

### Items
![image](https://github.com/kodadot/nft-gallery/assets/90852205/2aa82fe1-ee7e-4b13-ace9-e858d47f2596)
- new section, category
- the button are same as in the profile
![image](https://github.com/kodadot/nft-gallery/assets/90852205/c24274f7-645a-46fc-af58-93f0f18542bd)
- once applied, it creates breadcrumb in the breadcrumb section - category active
- if more than 1 category is applied, the breadcrumb changes to - categories active
- removing breadcrumb deselect all the categories

### Collection
- category filters will be placed in the section between the full width line and the divider - there is currently only collection counter
- mobile UI soon, probably will be solved with dropdown
![image](https://github.com/kodadot/nft-gallery/assets/90852205/f62ac2d4-bd5a-4e41-82d8-c5242256b647)

### Question
- on what chains we can use categories
- is it possible to set different category to a single nft placed in collection that already has category applied
- @vikiival sir 👀

**Comment by JustLuuuu** [comment]
also question, it will probably not be possible to change the category of existing NFTs?

**Comment by vikiival** [comment]
> on what chains we can use categories

`AH*`

> is it possible to set different category to a single nft placed in collection that already has category applied

Can you rephrase please?

> also question, it will probably not be possible to change the category of existing NFTs?

when we will have edit UI ;D

**Comment by exezbcz** [comment]
> Can you rephrase please?
> 
> >

so you are minting nft into collection - the collection already has a category. The question is whether you can set category for single items

**Comment by vikiival** [comment]
> The question is whether you can set category for single items

For Collectives it makes sense for sure. 

For normal collections: well I would then do collection category `mixed` and then you can set additional category per NFT

**Comment by exezbcz** [comment]
@shashkovdanil one
